### DISCUSSION TITLE
Debug test at current cursor position

### DISCUSSION THREAD

**Discussion by SkyCodeSky** [orta/vscode-jest#902] [Ideas]
Hi everyone. 😊 
Is there a feature to debug a test at the current cursor position?

<img width="200" alt="debug_at_cursor" src="https://user-images.githubusercontent.com/112804480/188329921-2ff712c9-5d3c-4260-9827-b12082a9f9a5.png">

- There is a test from line 3 to 5.
- Also there is an button (A) to run this test.
- Then, in test explorer, there is a button (B) to debug this very test, if you find it within the test explorer.
- It would be great if there would be a key shortcut (not mouse) to debug this very test if the cursor is in line 4 (C).

Is this a feature already? If so, what do I have to do? 😊 

All the best! 🤗

**Comment by connectdotz** [comment]
TestExplorer already has a shortcut for this: 
```json
{
  "key": "cmd+; cmd+c",
  "command": "testing.debugAtCursor",
  "when": "editorTextFocus"
}
```
does this work for you?

**Comment by SkyCodeSky** [comment]
Nice! 😊 Exactly what I was hoping for. It works great. Thanks a lot!
